src/components/Button/Button.tsx

```diff
@@ -1,8 +1,19 @@
+// hooks
```

Comment:
Please remove unnecessary comments. Only use them in situations that are hard to understand.